doc/tutorials/gpu-passthru.rst

```diff
@@ -61,6 +61,15 @@ The example extends the information provided in the :ref:`gsg`.
 #. In the ACRN Configurator, create a shared scenario with a Service VM and one
    post-launched User VM.
 
+#. Select the Service VM and go to **Basic Parameters > Kernel
+   command-line parameters**.
+
+#. Append ``i915.modeset=0`` in **Kernel command-line parameters**.
+
+   .. image:: images/kernel_cmd_line.png
```

Comment:
There should not be red squiggly underlines in the picture.  Is this an actual screen shot from the configurator?

Reply:
Yes, this is an actual screen shot from the configurator.

Reply:
Will update the picture.

Reply:
updated.